circle.yml

```diff
@@ -57,14 +57,14 @@ test:
     - touch $CIRCLE_TEST_REPORTS/xunit/server-tests.xml
   override:
     # Run client tests
-    - ./node_modules/.bin/mocha --timeout 30000 -R xunit dist/test.js > $CIRCLE_TEST_REPORTS/xunit/client-tests.xml:
+    - ./node_modules/.bin/mocha --timeout 90000 -R xunit dist/test.js > $CIRCLE_TEST_REPORTS/xunit/client-tests.xml:
         pwd: client
-        parallel: true
+        parallel: false
```

Comment:
I don't understand how CircleCI's parallelism works exactly, but it seemed to break the tests. Almost as if some of the `pre` steps weren't properly applied across all parallel instances.